### DISCUSSION TITLE
Popover component: Thoughts and advice

### DISCUSSION THREAD

**Discussion by kevh-c** [twilio-labs/paste#815] [Q&A] [answered]
Hi all — we're exploring using the popover component to provide more information about Conversations for users who aren't familiar with the product.

In this use case a user is within a Programmable Messaging service and will be exploring what they want their service to do with incoming messages. They may not have an understanding on what Conversations is so instead of forcing the user away to Conversations straight away, we are thinking of providing them a brief explanation on what the API offers with an explicit link to find out more. This link will take them to Conversations on Console.

Would love any thoughts or insight on using the popover component in this context.

![Integration 05](https://user-images.githubusercontent.com/2405518/94791331-6145a580-03cf-11eb-8627-7bc5abe1325b.png)

**Comment by kate-porter** [comment, marked as answer]
Hey @kevhealy-git - I think this looks great! This is a good use case for a popover. I like that the user doesn't get navigated to a new page right away, but instead gets a glimpse of the info in the popover first.

**Reply by kevh-c** [reply]
@katieporter Thanks Katie — we're intending to test this with a handful of customers soon so I will update on any feedback we get about this component in particular.

**Reply by kate-porter** [reply]
That would be great, thank you!

**Comment by SiTaggart** [comment]
Hi @kevhealy-git, quick clarifying question for this one; Is the intention that "Conversations" is _also_ still a link?

The reason I ask is that for a number of limitations and accessibility reasons, an element can't be two things at the same. An element on the whole can only perform **a single distinct action**; "I am a link to a new page", "I launch a modal", "I launch a popover".

If you are still intending to have the word conversations be a link to a new page, then you would need another mechanism to launch the popover.

If the word conversations _only_ launches the popover on click, then that's fine, the only difference being you'd be using a link style button, which is not underlined.

**Reply by kevh-c** [reply]
Hey Simon — no, "Conversations" would just trigger the popover and the link to a new page would be triggered by the "Find out more" link. Good to know about the link-style button. I will update the designs to reflect this.

**Reply by SiTaggart** [reply]
Perfect, thanks @kevhealy-git
